.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: node_js
 node_js:
   - node
-  - 4.4.5
+  - 5.11.1

```

Comment:
I noticed Heroku would not install 4.4.5 and always picked 5.11.1 - I thought v5 supported the spread operator but turns out only with `--harmony` set. NOPE.